innovation.game.php

```diff
@@ -1234,6 +1237,38 @@ function transferCardFromTo($card, $owner_to, $location_to, $bottom_to=false, $s
         }
         return $card;
     }
+
+    function setCardIconHash() {
+        //LMF: unsure if I have to grab this list from the db, I don't fully understand texual_card_infos.
```

Comment:
I think this approach is right. This comment is a question to someone more knowledgeable of this file.